src/validation.cpp

```diff
@@ -716,27 +716,22 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
                               fSpendsCoinbase, nSigOpsCost, lp);
         unsigned int nSize = entry.GetTxSize();
 
-        // Check that the transaction doesn't have an excessive number of
```

Comment:
Good question.

Reply:
I think this comment is not very helpful. It was originally added in #4150, and in the review on that PR people complained that the phrasing in this comment is confusing ("invalid rather than merely non-standard" - huh?).

If reviewers prefer it, then I can just improve this comment rather than delete it -- it just seems to me like the code reads just fine on its own.

Reply:
`MAX_BLOCK_SIGOPS` is replaced by `MAX_BLOCK_SIGOPS_COST` (which is just multiplied by the witness scale factor of 4) as part of segwit. `MAX_STANDARD_TX_SIGOPS_COST` is just a separate rule at the relay/mempool level saying "you have to use at least 5 tx's to hit the sigop limit".

I agree that the comment's just confusing given how we understand "invalid" (breaks consensus rules) vs "non-standard" (not interesting for mempool/relay, but not too strongly punished either).